docs/baking.md

```diff
@@ -0,0 +1,185 @@
+
+In order to run a baking instance, you'll need the following Tezos binaries:
+`tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
+
+Currently used proto's are `007-PsDELPH1` (used on mainnet and delphinet) and
+`008-PtEdoTez` (used on edonet). Also, note that the corresponding ubuntu packages have protocol
+suffix in lowercase, e.g. list of available baker packages can be found
+[here](https://launchpad.net/~serokell/+archive/ubuntu/tezos/+packages?field.name_filter=tezos-baker&field.status_filter=published).
+
+To install them run the following commands:
+```
+# Add PPA with Tezos binaries
+sudo add-apt-repository ppa:serokell/tezos
+sudo apt-get update
+# Install packages
+sudo apt-get install tezos-client tezos-node tezos-baker-<proto> tezos-endorser-<proto>
+```
+
+Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` provide
+systemd units for running the corresponding binaries in the background.
+
+### Downloading zcash-params
+
+Since v8.0 it's required to have the `sapling-output.params` and `sapling-spend.params` files from
+the `zcash-params` within the `XDG_DATA_DIRS` to run the `tezos-node`.
+
+We'll need these params to be accessible by different
+users (because `systemd` services are usually run as a separate user), thus the suggested way of getting
+them via [the zcash script](https://raw.githubusercontent.com/zcash/zcash/master/zcutil/fetch-params.sh), which will
+download them to the `HOME` directory of the current user, is not convenient. Moreover, it will
+download the large `sprout-groth16.params` which is unused at runtime.
+So, it's better to download them to the `/usr/local/share/zcash-params`, the easiest way to do that
+is to run [`fetch-sapling-params.sh`](../scripts/fetch-sapling-params.sh) script:
+```
+sudo ./fetch-sapling-params.sh
+```
+
+Sudo is required because the files are downloaded to the `/usr/local/share/zcash-params`.
+This script also checks that hashsums of the downloaded files match the expected values.
+
+## Setting up the tezos-node
+
+The easiest way to set up a node running on the `mainnet` or on one of the
+testnets (e.g. `delphinet` or `edonet`) is to use one of the predefined
+`tezos-node-<network>.services` systemd services provided in the `tezos-node`
+package. 
+
+However, by default, these services will start to bootstrap the node from scratch,
+which will take a significant amount of time.
+In order to avoid this, here we suggest bootstrapping from a snapshot instead.
+
+### Setting up node data directory
+
+`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
+and `http://localhost:8732` as its RPC address by default.
+These parameters can be changed in the service configuration, which is located in
+`/lib/systemd/system/tezos-node-<network>.service`. Note, that if you've updated the `.service`
+file, you should reload systemctl daemon via running `sudo systemctl daemon-reload`.
```

Comment:
users should be cautioned against editing .service files directly since they are part of the package; it [seems that current recommended way](https://serverfault.com/questions/413397/how-to-set-environment-variable-in-systemd-service?answertab=votes#tab-top) is to use `systemctl edit`

Reply:
it looks like this section doesn't actually set up anything, it's just information; would be good to explain why user may want/need to change the defaults